scss/_buttons.scss

```diff
@@ -4,7 +4,7 @@
 
 .btn {
   display: inline-block;
-  font-family: $btn-font-family;
+  font-family: if($btn-font-family != null, var(--btn-font-family), null);
```

Comment:
Why do we need this test?

Reply:
ok, I get it, nvm

Reply:
If we simply use `var()`, the property would be generated even if the Sass variable is `null` —thus not defining the custom property—which would defeat the whole point.

Am I wrong?